server/game/core/Constants.ts

```diff
@@ -306,4 +306,11 @@ export enum AbilityRestriction {
     Ready = 'ready',
     ReceiveDamage = 'receiveDamage',
     TriggerAbilities = 'triggerAbilities',
+}
+
+export enum ViewCardType {
```

Comment:
Two small comments:

1. Since this is only relevant for one situation we can probably move it to `ViewCardSystem.ts` to avoid cluttering this file too much
2. How would you feel about renaming it to `ViewCardMode` since (a) 'mode' is typically how a parameter like this is named elsewhere in the repo (e.g., `TargetMode`) and (b) 'CardType' here sounds like `CardType` as in unit, upgrade, event etc.